af/fastpath/fastpath/core.py

```diff
@@ -1080,6 +1080,7 @@ def score_psiphon(msm) -> dict:
         logbug(0, "no resolver_ip", msm)
         scores["accuracy"] = 0.0
 
+    scores["extra"] = dict(test_runtime=msm.get("test_runtime"))
```

Comment:
Psiphon also has the key `bootstrap_time` nested under `test_keys`. While we are at it, let's add support for extracting that as well.